spec/menu_spec.rb

```diff
@@ -0,0 +1,29 @@
+require './lib/menu'
+
+describe Menu do
+
+  describe '#initialize' do
+    it 'is initialized with an empty hash' do
+      menu = Menu.new
+      expect(menu.contents).to be_kind_of(Hash)
+    end
+  end
+
+  describe '#add_dish' do
+    it 'responds to add_dish' do
+      menu = Menu.new
+      expect(menu).to respond_to(:add_dish)
+    end
+    it 'adds a dish' do
+      menu = Menu.new
+      d1 = double('dish')
+      allow(d1).to receive_messages(
+      :name => "Chicken Korma",
```

Comment:
Minor — the indentation style seems slightly off in some places, did you try running rubocop over those files? 🤔